projects/packages/search/src/dashboard/components/record-meter/bar-chart.jsx

```diff
@@ -146,7 +146,11 @@ export class BarChart extends React.Component {
 									/>
 									<span className="jp-search-chart-legend__label" children={ item.text } />
 									<span className="jp-search-chart-legend__count">
-										({ this.props.data[ item.datasetIndex ].data.data?.toLocaleString() })
+										(
+										{ typeof this.props.data[ item.datasetIndex ]?.data?.data === 'object'
```

Comment:
On further inspection, it looks like `this.props.data[ item.datasetIndex ]?.data?.data` is an array with a single number at position `0`.

I've simplified this a bit and will push a commit.

Reply:
Updated in [cec942d](https://github.com/Automattic/jetpack/pull/23858/commits/cec942d89c68ffe7aceaf77f083f235e48383a54).